docs/csharp/language-reference/builtin-types/value-tuples.md

```diff
@@ -75,46 +80,52 @@ C# supports assignment between tuple types that satisfy both of the following co
 
 Tuple element values are assigned following the order of tuple elements. The names of tuple fields are ignored and not assigned, as the following example shows:
 
-[!code-csharp-interactive[tuple assignment](snippets/shared/ValueTuples.cs#Assignment)]
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="Assignment":::
 
-You can also use the assignment operator `=` to *deconstruct* a tuple instance in separate variables. You can do that in one of the following ways:
+You can also use the assignment operator `=` to *deconstruct* a tuple instance in separate variables. You can do that in many ways:
 
 - Explicitly declare the type of each variable inside parentheses:
 
-  [!code-csharp-interactive[specify types of variables](snippets/shared/ValueTuples.cs#DeconstructExplicit)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructExplicit":::
 
 - Use the `var` keyword outside the parentheses to declare implicitly typed variables and let the compiler infer their types:
 
-  [!code-csharp-interactive[implicitly typed variables](snippets/shared/ValueTuples.cs#DeconstructVar)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructVar":::
 
 - Use existing variables:
 
-  [!code-csharp-interactive[existing variables](snippets/shared/ValueTuples.cs#DeconstructExisting)]
+  :::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="DeconstructExisting":::
+
+Any of these can be combined in a single deconstruction expression.
+
+You can also combine deconstruction with [pattern matching](../../fundamentals/functional/pattern-matching.md) to inspect the characteristics of fields in a tuple. The following example loops through several integers and prints those that are divisible by 3. It deconstructs the tuple result of <xref:System.Int32.DivRem%2A?displayProperty=nameWithType> and matches against a `Remainder` of 0:
```

Comment:
To match the article text, the example should be updated as follows:
```csharp
if (Math.DivRem(i, 3) is ( Quotient: var q, Remainder: 0 ))
{
    Console.WriteLine($"{i} is divisible by 3, with quotient {q}");
}
```

Note parentheses instead of braces (that transforms property pattern to positional pattern) and swapped quotient and remainder.